src-backend/registration/tests/test_user.py

```diff
@@ -10,8 +10,5 @@ def setUp(self):
         self.test_user.save()
 
     def test_user_has_token(self):
-        try:
-            token = Token.objects.get(user=self.test_user)
-            self.assertFalse(token is None)
-        except ObjectDoesNotExist:
-            self.assertTrue(False)
+        token = Token.objects.get(user=self.test_user)
```

Comment:
http://stackoverflow.com/questions/7799593/how-an-exceptions-be-tested-with-nose